### PR TITLE
Fix comment page wrapping for non-logged in users

### LINKED ISSUE #8287: Sidebar wrap problem for non-logged-in users
I noticed that when not logged in, the sidebar gets shoved below the rest of content on note pages like this: https://publiclab.org/notes/anngneal/12-08-2017/7-ways-to-measure-monitor-and-evaluate-water-quality

image

Tracing it, I thought this is the juncture between divs that's relevant:

https://github.com/publiclab/plots2/blob/73193e0cfca8a648cc2f7db2a342db1197a11576/app/views/notes/show.html.erb#L110-L118

However, I think it's this line which has an unclosed div:

https://github.com/publiclab/plots2/blob/69b8a56ce8c170b74151debf36da57ae8dd762be/app/views/notes/_comment.html.erb#L79-L92

That would only be shown to logged-in users. For non-logged in users, we'd be left with an extra `</div>` on the lines below, causing this issue. Let's test in GitPod.

### PR DESCRIPTION
**Pull request by jywarren**
Fixes #8287

Testable on any notes page when not logged in, in full width display; 

image

### CODE REVIEW

**Comment by jywarren** [comment]
Hmm. It caused an error:

```

==[Screenshot]: tmp/screenshots/failures_test_adding_a_reply_comment_via_javascript_with_url_only.png
 FAIL["test_adding_a_reply_comment_via_javascript_with_url_only", #<Minitest::Reporters::Suite:0x00007f201c3d5d40 @name="CommentTest">, 79.86760069299999]
 test_adding_a_reply_comment_via_javascript_with_url_only#CommentTest (79.87s)
        expected to find css "#c474456746 .comment .comment-body p" but there were no matches
        test/system/comment_test.rb:47:in `block in <class:CommentTest>'
=[Screenshot]: tmp/screenshots/failures_test_comment_editing.png
ERROR["test_comment_editing", #<Minitest::Reporters::Suite:0x00007f2022a6ebb0 @name="CommentTest">, 145.51373539600002]
 test_comment_editing#CommentTest (145.52s)
Minitest::UnexpectedError:         Capybara::ElementNotFound: Unable to find css ".alert-success"
            test/system/comment_test.rb:234:in `block in <class:CommentTest>'
```

File: app/views/notes/_comment.html.erb [hunk 1]
```diff
@@ -85,6 +85,7 @@
                 <img alt="React to post" class="emoji" height="20" width="20" src="<%= emoji_image_map[e] %>">
               <% end %>
               <% end %>
+            </div>
```
Comment (jywarren) [inline app/views/notes/_comment.html.erb:88]: I think this is a close `<div>` in the wrong place, maybe... causing these errors:

```
        expected to find css "#c474456746 .comment .comment-body p" but there were no matches
Minitest::UnexpectedError:         Capybara::ElementNotFound: Unable to find css ".alert-success"
```

**Comment by jywarren** [comment]
I bet we could figure this out with some trial and error...

**Comment by cesswairimu** [comment]
Just realized  I undid your change on my commit, the divs seem to be in their right places. 
Checking on the sidebar section....I have a feeling the `login to comment` section might be the problem

**Comment by jywarren** [comment]
Somehow perhaps due to other changes, i believe this is resolved now! Thank you!!!

**Pull request closed without merging**